crates/ruff_python_formatter/src/options.rs

```diff
@@ -21,7 +23,9 @@ pub struct PyFormatOptions {
 
     /// The (minimum) Python version used to run the formatted code. This is used
     /// to determine the supported Python syntax.
-    target_version: PythonVersion,
+    #[serde(deserialize_with = "adapter::deserialize::<_, PythonVersion>")]
+    #[serde(serialize_with = "adapter::serialize::<_, PythonVersion>")]
```

Comment:
Is this type used anywhere in a user facing facade? If not, I then think it's fine if the representation changes.